docs/introduction.md

```diff
@@ -4,145 +4,62 @@
+
+- Application Centric. Behind the Appfile, KubeVela enforces the corresponding **application** concept as its main API and **ALL** KubeVela's capabilities serve for the applications' needs only. This is achieved by adopting the [Open Application Model](https://github.com/oam-dev/spec) as the main API for the platform.
  
-- A KubeVela application is composed of various supported components or
-  [traits](https://github.com/oam-dev/spec/blob/master/introduction.md) and their
-  schemas are determined by the platform builders. New capabilities can be added
-  to KubeVela through a CRD registry mechanism. 
-
-- KubeVela provides a rich set of tools to help platform builders to abstract
-  the Kubernetes resource and CRD APIs.
-  For example, a [CUELang](https://github.com/cuelang/cue) based templating tool
-  is used to easily build the contract between the user-facing schemas and the
-  underline Kubernetes Objects. KubeVela provides a set of built-in CUE templates
-  for platform builders to start with.
-
-- Schema changes take effect immediately. Neither recompiliation nor redeployment
-  of KubeVela is required. This makes the process of delivering new capabilities
-  using KubeVela extremely simple.
+- Natively Extensible. An application in KubeVela is composed of various pluggable workload types and operational traits. Capabilities from Kubernetes ecosystem can be added to KubeVela as new workload types or trait through Kubernetes CRD registry mechanism at any time.
+
+- Simple yet Extensible Abstraction Layer. KubeVela's main interfaces, Appfile and CLI, are built with [CUELang](https://github.com/cuelang/cue) as the abstraction layer between the user-facing schemas and the underline Kubernetes resources. KubeVela provides a set of built-in abstractions to start with and platform teams are free to modify them at any time. Abstraction changes take effect at runtime, neither recompilation nor redeployment of KubeVela is required. 
   
-With KubeVela, platform builders now have indefinite flexibilities in designing
-and implementing new capabilities without worrying about what and how to expose
-the new capabilities to the end users. However, the ultimate beneficiaries are
-the application developers:
-- Instead of managing a handful Kubernetes YAML files, only a simple
-  docker-compose style [appfile](./docs/developers/devex/appfile.md) is needed
-  to manage an application in KubeVela. 
-- Out of the box, KubeVela provides a CLI tool to simplify the application
-  management workflow which can be easily integrated with existing CI/CD pipelines.
+With KubeVela, platform builders now finally have the tooling support to design and ship any new capabilities to their end-users with high confidence and turn around time measured in hours. 
```

Comment:
- measured in hours ( is it good or not?      Without knowing the context, one cannot justify if it is impressive or not). So either choose even aggressive but intuitive expression say `measured in minutes` or just say `low turn around time`.